![Imgur](https://i.imgur.com/Mb7kt57.png)

#<font color = bluelight> Projeto Bad Cripton

## <font color = bluelight> >>Introdução

>    Nos últimos anos, as criptomoedas emergiram como uma classe de ativos revolucionários, desafiando as estruturas tradicionais de finanças e investimentos. Essas moedas digitais descentralizadas, baseadas na tecnologia de blockchain, oferecem uma série de vantagens, como a capacidade de realizar transações seguras e transparentes sem a necessidade de intermediários, como bancos. No entanto, essa inovação também trouxe desafios significativos, incluindo a sua alta volatilidade e impacto ambiental.
Em um mundo cada vez mais digital, as criptomoedas desempenham um papel fundamental na economia atual, atraindo investidores e especuladores de todo o mundo. Seu valor e popularidade cresceram exponencialmente, resultando em uma série histórica de preços que reflete a evolução dessas ações digitais.
Neste contexto, nossa equipe foi designada por uma corretora financeira para conduzir uma análise exploratória da série histórica dos valores das criptomoedas. O objetivo é compreender os melhores padrões e tendências por trás desses ativos, bem como avaliar seu impacto no mercado financeiro e na economia global.
Além disso, nossa análise também abordará uma questão crucial: a pegada de carbono das criptomoedas. A mineração de criptomoedas, em especial o Bitcoin, consome uma quantidade significativa de energia elétrica, levando a preocupações sobre seu impacto ambiental. Avaliar essa pegada de carbono é essencial para entender o equilíbrio entre a inovação financeira e as preocupações ambientais.
Ao longo deste estudo, examinaremos os dados históricos das principais criptomoedas, explorando suas variações de preço, volume de negociação e capitalização de mercado. Além disso, investigaremos as práticas de mineração de criptomoedas e os esforços em curso para tornar essa atividade mais sustentável.
Nossa análise exploratória visa fornecer uma visão abrangente das criptomoedas, abordando tanto os benefícios quanto os desafios associados a esses ativos digitais. Por meio de dados, insights e análises, esperamos contribuir para uma compreensão mais completa dessas questões financeiras e ambientalmente relevantes.

### <font color = bluelight> >>> Métodos de Mineração



> - Existem diferentes métodos de mineração de criptomoedas, incluindo Proof of Work (PoW) e Proof of Stake (PoS). A maioria das criptomoedas populares, como o Bitcoin, utiliza o PoW, que consome muita energia. Criptomoedas baseadas em PoS geralmente têm uma pegada de carbono menor.

## <font color = bluelight> >> Objetivos do Projeto
  >- Realizar a análise exploratória a partir das fontes de dados disponíveis no Kaggle
  >- Serão utilizados os arquivos que correspondem a uma serie histórica de criptomoedas (10 escolhidas)do ano de 2013 a 2021.
  >- Análise descritiva das características dos dados, registros, tipos de dados e valores;
  >- Limpar e tratar os dados;
  >- Salvar na Base de Dados Postgres;
  >- Importar em uma plataforma de vizualização (Tableau, BI ou Looker)
  >- Utilizar e visualizar gráficos com o objetivo de obter  insights.
  ---

### <font color = bluelight> >>> Perguntas a serem resolvidas

####>>>>   1. Como se comportaram os valores para todas as criptomoedas? Os valores tiveram uma tendência de queda ou de aumento?

- **R:** O comportamento dos valores para todas as criptomoedas foi variável ao longo do tempo, mas em geral houve uma tendência de aumento, especialmente nos últimos anos. Algumas moedas, como o bitcoin, o ethereum e o cardano, tiveram um crescimento mais expressivo do que outras, como o dogecoin, o stellar e o monero.

####>>>>  2. Quais os valores médios para todas as criptomoedas?

- **R:** Os valores médios para todas as criptomoedas foram calculados usando a média dos preços de fechamento de cada moeda ao longo do período analisado. Os valores médios foram os seguintes:
 - Moeda Valor médio (USD)
 - Bitcoin 7.438,77
 - Dogecoin 0,01
 - Litecoin 55,30
 - Solana 18,94
 - Ethereum 328,67
 - Stellar 0,11
 - Cosmos 5,86
 - Cardano 0,28
 - Monero 97,66
 - Polkadot 16,63

####>>>> 3. Em quais anos houve maiores quedas e valorizações?

- **R:** Os anos em que houve maiores quedas e valorizações dependem de cada moeda, mas alguns padrões podem ser observados. Por exemplo, o ano de 2017 foi marcado por uma alta valorização de várias moedas, como o bitcoin, o ethereum, o litecoin e o monero. Já o ano de 2018 foi caracterizado por uma forte queda de quase todas as moedas, com exceção do stellar e do cosmos. O ano de 2020 também foi um ano de recuperação para muitas moedas, especialmente o bitcoin e o ethereum. O ano de 2021 foi um ano de grande volatilidade, com altos e baixos para todas as moedas.

####>>>> 4. Existe alguma tendência de aumento ou queda dos valores pelo dia da semana?

 - **R:** Não há uma tendência clara de aumento ou queda dos valores pelo dia da semana, mas alguns dias parecem ter uma maior variação do que outros. Por exemplo, as segundas-feiras e as quintas-feiras costumam ter uma maior amplitude entre os preços máximos e mínimos do que os sábados e os domingos. Isso pode estar relacionado com a maior movimentação do mercado financeiro nesses dias.

####>>>>  5. Qual moeda se mostra mais interessante em relação à valorização pela análise da série histórica?

 - **R:** A moeda que se mostra mais interessante em relação à valorização pela análise da série histórica é o bitcoin, pois foi a que teve o maior crescimento absoluto e relativo ao longo do tempo. O bitcoin começou valendo menos de um dólar em 2010 e chegou a valer mais de 60 mil dólares em 2021, um aumento de mais de 6 milhões por cento. Além disso, o bitcoin é a moeda mais consolidada e reconhecida no mercado de criptomoedas, com a maior capitalização de mercado e liquidez.

####>>>>  6. Qual moeda se mostra menos interessante em relação à valorização pela análise da série histórica?

 - **R:** A moeda que se mostra menos interessante em relação à valorização pela análise da série histórica é o dogecoin, pois foi a que teve o menor crescimento absoluto e relativo ao longo do tempo. O dogecoin começou valendo menos de um centavo de dólar em 2013 e nunca ultrapassou a marca de um dólar até hoje, um aumento de cerca de 10 mil por cento. Além disso, o dogecoin é uma moeda que tem uma origem humorística e não tem um propósito claro ou uma vantagem competitiva no mercado de criptomoedas.

####>>>>7. Existe correlação entre os valores para todas as criptomoedas?

 - **R:** Existe correlação entre os valores para todas as criptomoedas, mas essa correlação varia em intensidade e direção dependendo das moedas comparadas. Mas, exemplo claro de correlação é o caso do bitcoin e do ethereum, que têm uma correlação positiva e forte de 0,88. Isso significa que os valores dessas duas moedas tendem a se mover na mesma direção e com uma magnitude similar. Ou seja, quando o bitcoin sobe, o ethereum também sobe, e quando o bitcoin cai, o ethereum também cai. Isso pode ser explicado pelo fato de que o bitcoin e o ethereum são as duas moedas mais populares e influentes do mercado de criptomoedas, e que muitos investidores e usuários as consideram como alternativas ou complementos entre si.

## <font color= bluelight>>> Importação de pacotes de bibliotecas que serão utilizados no projeto bem como suas versões

In [54]:
# Pacotes de bibliotecas que serão usadas no projeto:
import pandas as pd
import numpy as np
import os


# versões dos pacotes:
print('Pacotes Instalados! Versões Utilizadas:\n')
print(f'Pandas      --> {pd.__version__}')
print(f'Numpy       --> {np.__version__}')


Pacotes Instalados! Versões Utilizadas:

Pandas      --> 1.5.3
Numpy       --> 1.23.5


---

---
##<font color = bluelight> >>Conhecendo os Dados:


>- Nesta etapa do projeto vamos entender os dados iniciais, suas características, registros, tipos e valores, aplicando uma análise com as estatísticas descritivas.

### <font color = bluelight> >>>Coletando os dados iníciais


 - As fontes de dados estão disponíveis no site da Kaggle. Serão utilizados os arquivos das seguintes criptomoedas (10 criptos) link encontra se disponível logo abaixo.
 -bitcoin
 -cardano
 -cosmos
 -dogecoin
 -ethereum
 -litecoin
 -monero
 -polkadot
 -solana
 -stellar
  -  Link [Serie historica de valores de criptomoedas ](https://www.kaggle.com/datasets/sudalairajkumar/cryptocurrencypricehistory)
  - Siga a [Documentação](https://colab.research.google.com/drive/1sIQuGii297HHdrJJkP3qeqbGF55ydJ-S?usp=sharing) após o download

### <font color = bluelight>>>> Carregando a base de dados

####>>>> Importando os arquivos

In [55]:
from google.colab import files
uploaded = files.upload()

Saving tb_final.csv to tb_final (3).csv


In [56]:
df_final=pd.read_csv('tb_final.csv')

In [57]:
df_final.head()

name symbol        date        high         low        open       close  \
0  Bitcoin    BTC  2013-04-29  147.488007  134.000000  134.444000  144.539993   
1  Bitcoin    BTC  2013-04-30  146.929993  134.050003  144.000000  139.000000   
2  Bitcoin    BTC  2013-05-01  139.889999  107.720001  139.000000  116.989998   
3  Bitcoin    BTC  2013-05-02  125.599998   92.281898  116.379997  105.209999   
4  Bitcoin    BTC  2013-05-03  108.127998   79.099998  106.250000   97.750000   

   volume     marketcap  
0     0.0  1.603769e+09  
1     0.0  1.542813e+09  
2     0.0  1.298955e+09  
3     0.0  1.168517e+09  
4     0.0  1.085995e+09

In [58]:
df_final.shape

(19022, 9)

### <font color = bluelight>>>> Funções

####>>>> func_renomea_colunas

>- Função utiliza para renomera as colunas do dataframe de cada criptomoeda, com o objetivo de colocar no idioma da língua portuguêsa para um melhor entendimento.

In [59]:
# renomeando as siglas das regioes usando replace e visualizando
def func_renomea_colunas(df):
  df.columns=['Nome','Sigla','Data','Max_Dia','Min_Dia','Valor_abertura','Valor_fechamento','Volume_negociacao','Valor_Mercado']
  return df

---

####>>>> func_adicionando_colunas

> - Função utilizada para adicionar colunas que serão necessarias, para uma melhor análise dos dados, foi adicionado dias da semana, que serão usados para responder algumas perguntas norteadoras. Também foi ajustado o formato da data.

In [60]:
def func_adicionando_colunas(df):
# Certifique-se de que a coluna de data esteja no formato de data, se não estiver, converta-a
 df['Data'] = pd.to_datetime(df['Data'])


# Mapeamento personalizado dos nomes dos dias da semana em português
 dias_da_semana = {
    'Monday': 'Segunda-feira',
    'Tuesday': 'Terça-feira',
    'Wednesday': 'Quarta-feira',
    'Thursday': 'Quinta-feira',
    'Friday': 'Sexta-feira',
    'Saturday': 'Sábado',
    'Sunday': 'Domingo'
 }

# Use .dt.strftime('%A') para obter o dia da semana em inglês e, em seguida, aplique o mapeamento
 df['Dia_semana'] = df['Data'].dt.strftime('%A').map(dias_da_semana)
 df.insert(3, 'Dia_semana', df.pop('Dia_semana'))
 print(df[['Data', 'Dia_semana']])



---

####>>>> func_formata_data

>- Formatando a data para uma melhor visualização

In [61]:
def func_formata_data(df):
  df['Data'] = df['Data'].dt.strftime('%d/%m/%Y')
  return df

---

--> Para termos uma compreensão clara de nossos dados, iremos criar um dicionário de variáveis contendo as colunas do Dataset e a descrição de seu conteúdo:

- O conjunto de dados possui um arquivo CSV para cada moeda. O histórico de preços está disponível diariamente a partir de 28 de abril de 2013. Este conjunto de dados contém informações históricas de preços de algumas das principais criptomoedas por capitalização de mercado.
* > SNo: índex
* > Name: nome da criptomoeda
* > Symbol: simbolo da criptomoeda
* > Date: data da observação
* > Open: preço de abertura em determinado dia
* > High: preço mais alto em um determinado dia
* > Low: preço mais baixo em um determinado dia
* > Close: preço de fechamento em determinado dia
* > Volume: Volume de transações em um determinado dia
* > MarketCap: Capitalização de mercado em dólares americanos

---

### <font color = bluelight>>>>Descrevendo os dados


>-  Objetivo é identificar o formato, os tipos de variáveis e os aspctos gerais, para um melhor entendimento do problema.

####<font color = bluelight>>>>> Dataset

#####>>>>> Consultando o formato do dataset

######>>>>>> Shape

In [62]:
df_cripto =df_final
df_cripto

name symbol        date        high         low        open  \
0      Bitcoin    BTC  2013-04-29  147.488007  134.000000  134.444000   
1      Bitcoin    BTC  2013-04-30  146.929993  134.050003  144.000000   
2      Bitcoin    BTC  2013-05-01  139.889999  107.720001  139.000000   
3      Bitcoin    BTC  2013-05-02  125.599998   92.281898  116.379997   
4      Bitcoin    BTC  2013-05-03  108.127998   79.099998  106.250000   
...        ...    ...         ...         ...         ...         ...   
19017  Stellar    XLM  2021-07-02    0.273359    0.252874    0.269148   
19018  Stellar    XLM  2021-07-03    0.270331    0.259710    0.263304   
19019  Stellar    XLM  2021-07-04    0.272314    0.259118    0.264254   
19020  Stellar    XLM  2021-07-05    0.268272    0.252300    0.268272   
19021  Stellar    XLM  2021-07-06    0.267757    0.253547    0.253931   

            close        volume     marketcap  
0      144.539993  0.000000e+00  1.603769e+09  
1      139.000000  0.000000e+00  1.542813e+09  
2      116.989998  0.000000e+00  1.298955e+09  
3      105.209999  0.000000e+00  1.168517e+09  
4       97.750000  0.000000e+00  1.085995e+09  
...           ...           ...           ...  
19017    0.263427  3.487793e+08  6.118631e+09  
19018    0.264324  3.310069e+08  6.139464e+09  
19019    0.268128  3.376196e+08  6.227819e+09  
19020    0.254456  3.359242e+08  5.910264e+09  
19021    0.260190  3.604261e+08  6.049985e+09  

[19022 rows x 9 columns]

> Aqui temos a informçãp que o dataset df_cripto possui:
- 19022 --> linhas (registros)
- 10    --> colunas

######>>>>>> Tipo de Dados


>- é importante verificarmos os tipos das variáveis que integram a base: inteiros, floats, objetos. Essa análise indica na fase de preparação dos dados a necessidade de transformações.

In [63]:
# Ter a informaçao se existe registros nulos
df_cripto.info()


<class 'pandas.core.frame.DataFrame'>
RangeIndex: 19022 entries, 0 to 19021
Data columns (total 9 columns):
 #   Column     Non-Null Count  Dtype  
---  ------     --------------  -----  
 0   name       19022 non-null  object 
 1   symbol     19022 non-null  object 
 2   date       19022 non-null  object 
 3   high       19022 non-null  float64
 4   low        19022 non-null  float64
 5   open       19022 non-null  float64
 6   close      19022 non-null  float64
 7   volume     19022 non-null  float64
 8   marketcap  19022 non-null  float64
dtypes: float64(6), object(3)
memory usage: 1.3+ MB


>--> Em df_cripto temos float64(6), object(3), int64(1)
- coluna 'Data' talves precise ser converitida para date


######>>>>>> Nulos

In [64]:
# Verificando quantidade de valores nulos
df_cripto.isnull().sum()

name         0
symbol       0
date         0
high         0
low          0
open         0
close        0
volume       0
marketcap    0
dtype: int64

--> Na analise de df_cripto, não temos valores nulos em nenhuma coluna.

##### >>>>> Limpando os Dados


---

> - Objetivo é identificar o que será necessário no projeto, para uma análise precisa, vamos retirar informações que não serão usadas ou que atrapalhem na vizualização dos insights.

######>>>>>> Dropando

In [65]:
df_cripto.head()

name symbol        date        high         low        open       close  \
0  Bitcoin    BTC  2013-04-29  147.488007  134.000000  134.444000  144.539993   
1  Bitcoin    BTC  2013-04-30  146.929993  134.050003  144.000000  139.000000   
2  Bitcoin    BTC  2013-05-01  139.889999  107.720001  139.000000  116.989998   
3  Bitcoin    BTC  2013-05-02  125.599998   92.281898  116.379997  105.209999   
4  Bitcoin    BTC  2013-05-03  108.127998   79.099998  106.250000   97.750000   

   volume     marketcap  
0     0.0  1.603769e+09  
1     0.0  1.542813e+09  
2     0.0  1.298955e+09  
3     0.0  1.168517e+09  
4     0.0  1.085995e+09

#####>>>>> Ajustando os Dados

######>>>>>> Renomeando Colunas

> - Renomeando as colunas para um melhor entendimento

In [66]:
# renomeando as siglas das regioes usando replace e visualizando
func_renomea_colunas(df_cripto)

Nome Sigla        Data     Max_Dia     Min_Dia  Valor_abertura  \
0      Bitcoin   BTC  2013-04-29  147.488007  134.000000      134.444000   
1      Bitcoin   BTC  2013-04-30  146.929993  134.050003      144.000000   
2      Bitcoin   BTC  2013-05-01  139.889999  107.720001      139.000000   
3      Bitcoin   BTC  2013-05-02  125.599998   92.281898      116.379997   
4      Bitcoin   BTC  2013-05-03  108.127998   79.099998      106.250000   
...        ...   ...         ...         ...         ...             ...   
19017  Stellar   XLM  2021-07-02    0.273359    0.252874        0.269148   
19018  Stellar   XLM  2021-07-03    0.270331    0.259710        0.263304   
19019  Stellar   XLM  2021-07-04    0.272314    0.259118        0.264254   
19020  Stellar   XLM  2021-07-05    0.268272    0.252300        0.268272   
19021  Stellar   XLM  2021-07-06    0.267757    0.253547        0.253931   

       Valor_fechamento  Volume_negociacao  Valor_Mercado  
0            144.539993       0.000000e+00   1.603769e+09  
1            139.000000       0.000000e+00   1.542813e+09  
2            116.989998       0.000000e+00   1.298955e+09  
3            105.209999       0.000000e+00   1.168517e+09  
4             97.750000       0.000000e+00   1.085995e+09  
...                 ...                ...            ...  
19017          0.263427       3.487793e+08   6.118631e+09  
19018          0.264324       3.310069e+08   6.139464e+09  
19019          0.268128       3.376196e+08   6.227819e+09  
19020          0.254456       3.359242e+08   5.910264e+09  
19021          0.260190       3.604261e+08   6.049985e+09  

[19022 rows x 9 columns]

######>>>>>> Adicionando Colunas

> - Colunas adicionadas, para uma melhor análise dos dados, ajudando a responder perguntas norteadoras

In [67]:
func_adicionando_colunas(df_cripto)

            Data     Dia_semana
0     2013-04-29  Segunda-feira
1     2013-04-30    Terça-feira
2     2013-05-01   Quarta-feira
3     2013-05-02   Quinta-feira
4     2013-05-03    Sexta-feira
...          ...            ...
19017 2021-07-02    Sexta-feira
19018 2021-07-03         Sábado
19019 2021-07-04        Domingo
19020 2021-07-05  Segunda-feira
19021 2021-07-06    Terça-feira

[19022 rows x 2 columns]


> - Visualização após o ajuste da coluna

######>>>>>> Foramtando Data

In [68]:
func_formata_data(df_cripto)

Nome Sigla        Data     Dia_semana     Max_Dia     Min_Dia  \
0      Bitcoin   BTC  29/04/2013  Segunda-feira  147.488007  134.000000   
1      Bitcoin   BTC  30/04/2013    Terça-feira  146.929993  134.050003   
2      Bitcoin   BTC  01/05/2013   Quarta-feira  139.889999  107.720001   
3      Bitcoin   BTC  02/05/2013   Quinta-feira  125.599998   92.281898   
4      Bitcoin   BTC  03/05/2013    Sexta-feira  108.127998   79.099998   
...        ...   ...         ...            ...         ...         ...   
19017  Stellar   XLM  02/07/2021    Sexta-feira    0.273359    0.252874   
19018  Stellar   XLM  03/07/2021         Sábado    0.270331    0.259710   
19019  Stellar   XLM  04/07/2021        Domingo    0.272314    0.259118   
19020  Stellar   XLM  05/07/2021  Segunda-feira    0.268272    0.252300   
19021  Stellar   XLM  06/07/2021    Terça-feira    0.267757    0.253547   

       Valor_abertura  Valor_fechamento  Volume_negociacao  Valor_Mercado  
0          134.444000        144.539993       0.000000e+00   1.603769e+09  
1          144.000000        139.000000       0.000000e+00   1.542813e+09  
2          139.000000        116.989998       0.000000e+00   1.298955e+09  
3          116.379997        105.209999       0.000000e+00   1.168517e+09  
4          106.250000         97.750000       0.000000e+00   1.085995e+09  
...               ...               ...                ...            ...  
19017        0.269148          0.263427       3.487793e+08   6.118631e+09  
19018        0.263304          0.264324       3.310069e+08   6.139464e+09  
19019        0.264254          0.268128       3.376196e+08   6.227819e+09  
19020        0.268272          0.254456       3.359242e+08   5.910264e+09  
19021        0.253931          0.260190       3.604261e+08   6.049985e+09  

[19022 rows x 10 columns]

###>>> Exportando os Dados .csv

In [69]:
# Salve o DataFrame como um arquivo CSV temporário no ambiente do Colab.
temp_csv = 'tb_cripto.csv'
df_cripto.to_csv(temp_csv, index=False)

# Crie um link direto para download do arquivo CSV
from google.colab import files
files.download(temp_csv)

<IPython.core.display.Javascript object>

<IPython.core.display.Javascript object>

###<font color = bluelight>>>> Bitcoin

#####>>>>> A mineração no Bitcoin é o processo pelo qual novas unidades da criptomoeda Bitcoin (BTC) são criadas e transações são verificadas na rede Bitcoin. O Bitcoin utiliza um algoritmo de mineração chamado SHA-256 (Algoritmo de Hash Seguro 256 bits), que é projetado para ser altamente seguro e resistente a ataques.
Prova de Trabalho (PoW): O Bitcoin também utiliza um sistema de prova de trabalho para validar transações e criar novas moedas. Os mineradores de Bitcoin resolvem problemas de hash criptográfico extremamente complexos usando seus dispositivos de mineração, como ASICs (Application-Specific Integrated Circuits), para competir pela recompensa de bloco. Isso garante a segurança da rede e a descentralização do processo de mineração.

####>>>> Dados Estatísticos

>- Valor de Fechamento

In [70]:

# Crie um novo DataFrame contendo apenas os dados do Bitcoin
df_bitcoin = df_cripto[df_cripto['Nome'] == 'Bitcoin'].copy()

# Visualize as primeiras linhas do novo DataFrame para verificar
print(df_bitcoin.head())


      Nome Sigla        Data     Dia_semana     Max_Dia     Min_Dia  \
0  Bitcoin   BTC  29/04/2013  Segunda-feira  147.488007  134.000000   
1  Bitcoin   BTC  30/04/2013    Terça-feira  146.929993  134.050003   
2  Bitcoin   BTC  01/05/2013   Quarta-feira  139.889999  107.720001   
3  Bitcoin   BTC  02/05/2013   Quinta-feira  125.599998   92.281898   
4  Bitcoin   BTC  03/05/2013    Sexta-feira  108.127998   79.099998   

   Valor_abertura  Valor_fechamento  Volume_negociacao  Valor_Mercado  
0      134.444000        144.539993                0.0   1.603769e+09  
1      144.000000        139.000000                0.0   1.542813e+09  
2      139.000000        116.989998                0.0   1.298955e+09  
3      116.379997        105.209999                0.0   1.168517e+09  
4      106.250000         97.750000                0.0   1.085995e+09  


>- Valor de Fechamento

In [71]:
# Transforme a coluna 'Close' em uma string
df_bitcoin['Valor_fechamento'].astype(str).describe()


count      2991
unique     2978
top       129.0
freq          3
Name: Valor_fechamento, dtype: object

>--> No df_bitcoin foram registrados 2991 valores, com 2978 valores diferentes

In [72]:
df_bitcoin['Valor_fechamento'].describe()

count     2991.000000
mean      6711.290443
std      11298.141921
min         68.431000
25%        430.569489
50%       2286.409912
75%       8576.238715
max      63503.457930
Name: Valor_fechamento, dtype: float64

![descrição da imagem, dashboard construindo em powerbi sobre a moeda bitcoin com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/bit1.png?raw=true)

>- Média de valor de fechamento teve um aumento de 15.926% e média de valor de abertura teve um aumento de 15.878%, aumentaram em tendência entre 2014 e 2021
>- As maiores médias do valor minímo e do valor máximo foram em 2021(234,64 e 262,21).

> Insight:

> O Bitcoin, sendo a primeira criptomoeda, apresentou um desempenho impressionante. De 2014 a 2021, o BTC experimentou um aumento notável de mais de 14.000% na média de valor de fechamento, passando de cerca de $318 para mais de $47.000. Esse crescimento consistente consolidou o Bitcoin como a principal criptomoeda do mercado.

####>>>>Tendência

![](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/bit2.png?raw=true)

###<font color = bluelight>>>> Cardano

#####>>>>> A mineração no Cardano é o processo pelo qual novas unidades da criptomoeda Cardano (ADA) são criadas e transações são verificadas na rede Cardano. O Cardano utiliza um algoritmo de mineração chamado Ouroboros, que foi projetado para ser altamente seguro e eficiente.
Prova de Participação (PoS): O Cardano adota um sistema de prova de participação para validar transações e criar novas moedas. Os participantes, chamados de "stakers", mantêm e delegam ADA em suas carteiras para participar do processo de validação de blocos. Isso elimina a necessidade de mineração intensiva em energia, como ocorre com o Bitcoin, tornando o Cardano mais ecológico e eficiente em termos energéticos. Além disso, o sistema PoS promove a descentralização da rede.
Devido à sua abordagem de PoS, o Cardano é considerado mais eficiente em termos de consumo de energia e tem um impacto de carbono significativamente menor do que as criptomoedas baseadas em PoW.

####>>>> Dados Estatísticos

>- Valor de Fechamento

In [73]:

# Crie um novo DataFrame contendo apenas os dados do Cardano
df_cardano = df_cripto[df_cripto['Nome'] == 'Cardano'].copy()

# Visualize as primeiras linhas do novo DataFrame para verificar
print(df_cardano.head())


         Nome Sigla        Data     Dia_semana   Max_Dia   Min_Dia  \
2251  Cardano   ADA  05/09/2018   Quarta-feira  0.106217  0.087569   
2798  Cardano   ADA  06/09/2018   Quinta-feira  0.089429  0.083130   
2864  Cardano   ADA  07/09/2018    Sexta-feira  0.089645  0.083456   
2930  Cardano   ADA  08/09/2018         Sábado  0.085130  0.077505   
2995  Cardano   ADA  02/10/2017  Segunda-feira  0.030088  0.019969   

      Valor_abertura  Valor_fechamento  Volume_negociacao  Valor_Mercado  
2251        0.105267          0.087569         98708900.0   2.270418e+09  
2798        0.087525          0.088666         73239200.0   2.298847e+09  
2864        0.088521          0.083983         41868100.0   2.177433e+09  
2930        0.084120          0.078235         35208700.0   2.028397e+09  
2995        0.024607          0.025932         57641300.0   6.288991e+08  


>- Valor de Fechamento

In [74]:
# Transforme a coluna 'Close' em uma string
df_cardano['Valor_fechamento'].astype(str).describe()


count          1374
unique         1374
top       0.0875694
freq              1
Name: Valor_fechamento, dtype: object

In [75]:
# Transforme a coluna 'Close' em uma string
df_cardano['Valor_fechamento'].astype(str).describe()

count          1374
unique         1374
top       0.0875694
freq              1
Name: Valor_fechamento, dtype: object

>--> No df_cardano foram registrados 1374 valores, com 1374 valores diferentes

In [76]:
df_cardano['Valor_fechamento'].describe()

count    1374.000000
mean        0.256313
std         0.409691
min         0.018539
25%         0.045947
50%         0.087002
75%         0.183379
max         2.309113
Name: Valor_fechamento, dtype: float64

![descrição da imagem, dashboard construindo em powerbi sobre a moeda cardano com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/cardano1.png?raw=true)

>- Média de valor de fechamento teve um aumento de 15.926% e média de valor de abertura teve um aumento de 15.878%, aumentaram em tendência entre 2014 e 2021
>- As maiores médias do valor minímo e do valor máximo foram em 2021(234,64 e 262,21).

> Insight:

> Cardano, uma criptomoeda com foco em sustentabilidade e escalabilidade, mostrou um desempenho notável entre 2017 e 2021. Nesse período, ADA teve um aumento de aproximadamente 4.600% na média de valor de fechamento, demonstrando sua crescente importância no ecossistema cripto.

####>>>>Tendência

![descrição da imagem, dashboard construindo em powerbi sobre a moeda bitcoin com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/cardano2.png?raw=true)

###<font color = bluelight>>>> Cosmos

#####>>>>> A mineração no Cosmos é o processo pelo qual novas unidades da criptomoeda Cosmos (ATOM) são criadas e transações são verificadas na rede Cosmos. O Cosmos utiliza um algoritmo de consenso chamado Tendermint, que combina características de Proof of Stake (PoS) com alguns elementos de Proof of Work (PoW). Isso tem implicações na pegada de carbono do Cosmos.
Prova de Participação (PoS): O Cosmos adota um sistema de prova de participação para validar transações e criar novas moedas. Os participantes, conhecidos como "stakers" ou validadores, mantêm e delegam ATOM em suas carteiras para participar do processo de validação de blocos. Isso elimina a necessidade de mineração intensiva em energia, como ocorre com o Bitcoin, tornando o Cosmos mais ecológico e eficiente em termos energéticos. Além disso, o sistema PoS promove a descentralização da rede Cosmos, permitindo que os detentores de ATOM participem ativamente na segurança e operação da rede.

####>>>> Dados Estatísticos

>- Valor de Fechamento

In [77]:

# Crie um novo DataFrame contendo apenas os dados do Cardano
df_cosmos = df_cripto[df_cripto['Nome'] == 'Cosmos'].copy()

# Visualize as primeiras linhas do novo DataFrame para verificar
print(df_cosmos.head())


        Nome Sigla        Data     Dia_semana   Max_Dia   Min_Dia  \
4368  Cosmos  ATOM  15/03/2019    Sexta-feira  7.715249  6.432468   
4369  Cosmos  ATOM  16/03/2019         Sábado  8.305615  6.694531   
4370  Cosmos  ATOM  17/03/2019        Domingo  7.357443  4.727895   
4371  Cosmos  ATOM  18/03/2019  Segunda-feira  5.229982  4.828242   
4372  Cosmos  ATOM  19/03/2019    Terça-feira  5.206172  4.794501   

      Valor_abertura  Valor_fechamento  Volume_negociacao  Valor_Mercado  
4368        6.633174          7.504351       6.057301e+06            0.0  
4369        7.507990          7.383882       3.477393e+06            0.0  
4370        7.357443          4.776164       2.653565e+06            0.0  
4371        4.828242          5.110341       2.567201e+06            0.0  
4372        5.099978          4.821883       3.891084e+06            0.0  


>- Valor de Fechamento

In [78]:
# Transforme a coluna 'Close' em uma string
df_cosmos['Valor_fechamento'].astype(str).describe()


count               845
unique              845
top       7.50435129868
freq                  1
Name: Valor_fechamento, dtype: object

In [79]:
# Transforme a coluna 'Close' em uma string
df_cosmos['Valor_fechamento'].astype(str).describe()

count               845
unique              845
top       7.50435129868
freq                  1
Name: Valor_fechamento, dtype: object

>--> No df_cosmos foram registrados 845 valores, com 845 valores diferentes

In [80]:
df_cosmos['Valor_fechamento'].describe()

count    845.000000
mean       6.768099
std        5.936096
min        1.649203
25%        3.346352
50%        4.480583
75%        6.221525
max       29.435150
Name: Valor_fechamento, dtype: float64

![descrição da imagem, dashboard construindo em powerbi sobre a moeda bitcoin com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/cosmos1.png?raw=true)

>- Média de valor de fechamento teve um aumento de 15.926% e média de valor de abertura teve um aumento de 15.878%, aumentaram em tendência entre 2014 e 2021
>- As maiores médias do valor minímo e do valor máximo foram em 2021(234,64 e 262,21).

> Insight:

> Cosmos é outra criptomoeda com desempenho sólido. De 2017 a 2021, o ATOM viu um crescimento de mais de 2.500% na média de valor de fechamento, refletindo a confiança na tecnologia de cadeias de blocos interoperáveis.

####>>>>Tendência

![descrição da imagem, dashboard construindo em powerbi sobre a moeda bitcoin com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/cosmos2.png?raw=true)

###<font color = bluelight>>>> Dogecoin

#####>>>>>Prova de Trabalho (PoW): O Dogecoin utiliza um algoritmo de mineração baseado em Prova de Trabalho, semelhante ao Bitcoin. Isso significa que os mineradores competem para resolver problemas computacionais complexos e, como recompensa, receber novas unidades de Dogecoin e taxas de transação. A mineração PoW é conhecida por consumir grandes quantidades de energia elétrica.

####>>>> Dados Estatísticos

>- Valor de Fechamento

In [81]:

# Crie um novo DataFrame contendo apenas os dados do Litecoin
df_dogecoin = df_cripto[df_cripto['Nome'] == 'Dogecoin'].copy()

# Visualize as primeiras linhas do novo DataFrame para verificar
print(df_dogecoin.head())


          Nome Sigla        Data     Dia_semana   Max_Dia   Min_Dia  \
5220  Dogecoin  DOGE  16/12/2013  Segunda-feira  0.000866  0.000150   
5221  Dogecoin  DOGE  17/12/2013    Terça-feira  0.000289  0.000116   
5222  Dogecoin  DOGE  18/12/2013   Quarta-feira  0.000362  0.000205   
5223  Dogecoin  DOGE  19/12/2013   Quinta-feira  0.001520  0.000328   
5224  Dogecoin  DOGE  20/12/2013    Sexta-feira  0.001143  0.000662   

      Valor_abertura  Valor_fechamento  Volume_negociacao  Valor_Mercado  
5220        0.000299          0.000205                0.0   1.509085e+06  
5221        0.000207          0.000269                0.0   2.169688e+06  
5222        0.000267          0.000362                0.0   3.188943e+06  
5223        0.000395          0.001162                0.0   1.115034e+07  
5224        0.001143          0.000704                0.0   7.284337e+06  


>- Valor de Fechamento

In [82]:
# Transforme a coluna 'Close' em uma string
df_dogecoin['Valor_fechamento'].astype(str).describe()


count                   2760
unique                  2755
top       0.0002093920047627
freq                       2
Name: Valor_fechamento, dtype: object

>--> No df_dogecoin foram registrados 2760 valores, com 2755 valores diferentes

In [83]:
df_dogecoin['Valor_fechamento'].describe()

count    2760.000000
mean        0.013763
std         0.062559
min         0.000087
25%         0.000226
50%         0.001844
75%         0.002899
max         0.684777
Name: Valor_fechamento, dtype: float64

![descrição da imagem, dashboard construindo em powerbi sobre a moeda bitcoin com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/doge1.png?raw=true)

>- Média de valor de fechamento teve um aumento de 562,50% aumentando em tendência 2018 e 2021
>- As maiores médias do valor minímo e do valor máximo foram em 2021(0,16 e 0,20).

> Insight:

> Inicialmente lançada como uma criptomoeda meme em 2013, o Dogecoin tinha um valor extremamente baixo, muitas vezes avaliado em frações de centavos.
No entanto percebemos que em 2021 obteve um crscimento muito significativo.

>Hipotese:
Suponhamos que o Dogecoin ganhou popularidade em 2021, impulsionado por eventos como a promoção de celebridades e a atenção da mídia. Isso levou a um aumento substancial no valor de fechamento, atingindo níveis muito mais altos do que sua histórica faixa de preço. Em maio de 2021, o Dogecoin atingiu seu valor mais alto até então, mas manteve sua volatilidade característica.



####>>>>Tendência

![descrição da imagem, dashboard construindo em powerbi sobre a moeda bitcoin com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/doge2.png?raw=true)

###<font color = bluelight>>>> Ethereum

#####>>>>> O Ethereum foi proposto em 2013 e lançado em 2015 por Vitalik Buterin e outros desenvolvedores. A Fundação Ethereum desempenhou um papel fundamental no desenvolvimento e na promoção da plataforma.
O Ethereum foi o epicentro do boom das Finanças Descentralizadas (DeFi) e dos Tokens Não Fungíveis (NFTs). Plataformas DeFi, como Uniswap e Compound, e mercados NFT, como CryptoKitties e NBA Top Shot, se tornaram populares na blockchain Ethereum.

No PoW, o Ethereum usava o algoritmo Ethash, que envolve a mineração por meio de poder computacional para validar transações e adicionar blocos à blockchain. No entanto, devido a preocupações com o consumo de energia e escalabilidade, o Ethereum estava migrando para o PoS com a atualização do Ethereum 2.0.

Com o PoS, a validação de transações e a criação de novos blocos dependem do "staking" de tokens Ether (ETH) pelos participantes da rede, chamados de validadores. Esse novo algoritmo de consenso, chamado de "Casper," visa tornar a rede mais eficiente e sustentável, ao mesmo tempo em que aumenta a segurança.

####>>>> Dados Estatísticos

>- Valor de Fechamento

In [84]:

# Crie um novo DataFrame contendo apenas os dados do Ethereum
df_ethereum = df_cripto[df_cripto['Nome'] == 'Ethereum'].copy()

# Visualize as primeiras linhas do novo DataFrame para verificar
print(df_ethereum.head())


          Nome Sigla        Data     Dia_semana   Max_Dia   Min_Dia  \
7991  Ethereum   ETH  08/08/2015         Sábado  2.798810  0.714725   
7992  Ethereum   ETH  09/08/2015        Domingo  0.879810  0.629191   
7993  Ethereum   ETH  10/08/2015  Segunda-feira  0.729854  0.636546   
7994  Ethereum   ETH  11/08/2015    Terça-feira  1.131410  0.663235   
7995  Ethereum   ETH  12/08/2015   Quarta-feira  1.289940  0.883608   

      Valor_abertura  Valor_fechamento  Volume_negociacao  Valor_Mercado  
7991        2.793760          0.753325           674188.0   4.548689e+07  
7992        0.706136          0.701897           532170.0   4.239957e+07  
7993        0.713989          0.708448           405283.0   4.281836e+07  
7994        0.708087          1.067860          1463100.0   6.456929e+07  
7995        1.058750          1.217440          2150620.0   7.364501e+07  


>- Valor de Fechamento

In [85]:
# Transforme a coluna 'Close' em uma string
df_ethereum['Valor_fechamento'].astype(str).describe()

count                   2160
unique                  2158
top       11.951600074768066
freq                       2
Name: Valor_fechamento, dtype: object

In [86]:
# Transforme a coluna 'Close' em uma string
df_ethereum['Valor_fechamento'].astype(str).describe()

count                   2160
unique                  2158
top       11.951600074768066
freq                       2
Name: Valor_fechamento, dtype: object

>--> No df_ethereum foram registrados 2160 valores, com 2158 valores diferentes

In [87]:
df_ethereum['Valor_fechamento'].describe()

count    2160.000000
mean      383.910691
std       601.078766
min         0.434829
25%        13.819200
50%       198.643691
75%       386.435272
max      4168.701049
Name: Valor_fechamento, dtype: float64

![descrição da imagem, dashboard construindo em powerbi sobre a moeda bitcoin com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/ethereum1.png?raw=true)

>- Média de valor de fechamento teve um aumento de 15.926% e média de valor de abertura teve um aumento de 15.878%, aumentaram em tendência entre 2014 e 2021
>- As maiores médias do valor minímo e do valor máximo foram em 2021(234,64 e 262,21).

> Insight:

> O Ethereum, com sua funcionalidade de contratos inteligentes, é uma das criptomoedas mais importantes. De 2015 a 2021, o ETH teve um aumento impressionante de cerca de 22.000% na média de valor de fechamento, refletindo seu papel vital na evolução do espaço cripto.

####>>>>Tendência

![descrição da imagem, dashboard construindo em powerbi sobre a moeda bitcoin com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/ethereum2.png?raw=true)

###<font color = bluelight>>>> Litecoin

#####>>>>> Litecoin (LTC): O Litecoin é uma criptomoeda semelhante ao Bitcoin,também conhecido como a "prata" do Bitcoin, mas com tempos de confirmação de transação mais rápidos. Também utiliza o PoW como método de consenso.


####>>>> Dados Estatísticos

>- Valor de Fechamento

In [88]:

# Crie um novo DataFrame contendo apenas os dados do Litecoin
df_litecoin = df_cripto[df_cripto['Nome'] == 'Litecoin'].copy()

# Visualize as primeiras linhas do novo DataFrame para verificar
print(df_litecoin.head())


           Nome Sigla        Data     Dia_semana  Max_Dia  Min_Dia  \
10163  Litecoin   LTC  29/04/2013  Segunda-feira  4.57360  4.22564   
10164  Litecoin   LTC  30/04/2013    Terça-feira  4.57238  4.16896   
10165  Litecoin   LTC  01/05/2013   Quarta-feira  4.35686  3.52029   
10166  Litecoin   LTC  02/05/2013   Quinta-feira  4.03930  3.00717   
10167  Litecoin   LTC  03/05/2013    Sexta-feira  3.45361  2.39594   

       Valor_abertura  Valor_fechamento  Volume_negociacao  Valor_Mercado  
10163         4.36676           4.38390                0.0   7.538896e+07  
10164         4.40352           4.29649                0.0   7.402092e+07  
10165         4.28954           3.80101                0.0   6.560460e+07  
10166         3.78002           3.37198                0.0   5.828798e+07  
10167         3.39044           3.04491                0.0   5.269485e+07  


>- Valor de Fechamento

In [89]:
# Transforme a coluna 'Close' em uma string
df_litecoin['Valor_fechamento'].astype(str).describe()


count                   2991
unique                  2988
top       2.8446500301361084
freq                       2
Name: Valor_fechamento, dtype: object

>--> No df_litecoin foram registrados 2991 valores, com 2988 valores diferentes

In [90]:
df_litecoin['Valor_fechamento'].describe()

count    2991.000000
mean       49.279008
std        63.240464
min         1.157010
25%         3.794135
50%        29.900200
75%        62.025043
max       386.450779
Name: Valor_fechamento, dtype: float64

![descrição da imagem, dashboard construindo em powerbi sobre a moeda bitcoin com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/litecoin1.png?raw=true)

>- Média de valor de abertura teve um aumento de 2.893% e média de valor de fechamento teve um aumento de 2.855%, aumentaram em tendência entre 2013 e 2021
>- As maiores médias do valor minímo e do valor máximo foram em 2021(185,48 e 207,36).

> Insight:

> O Litecoin apresentou um impressionante aumento de 2.893% na média de valor de abertura e na média de valor de fechamento entre 2013 e 2021, indicando uma tendência de crescimento consistente. Além disso, em 2021, registrou suas maiores médias de valor mínimo (185,48) e valor máximo (207,36), o que sugere um desempenho notável nesse ano.

####>>>>Tendência

![descrição da imagem, dashboard construindo em powerbi sobre a moeda bitcoin com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/litecoin2.png?raw=true)

> Ao obeservar o gráfico identificamos que os anos em que o Litecoin obteve suas maiores quedas foi em 2015 com uma baixa de -72,63% em relação a 2014 e 2019 com uma baixa de -34,14 em relação a 2018.
> As maiores altas foram em 2017  com uma alta de 1220 % em relação a 2016 e 2021 com uma alta de 250,82% em relação a 2020.( obeservação não temos os dados completos do ano de 2021). Informações mais detalhadas no dashboard.

###<font color = bluelight>>>> Monero

#####>>>>> A mineração no Monero é o processo pelas quais novas unidades da criptomoeda Monero (XMR) são criadas e transações são verificadas na rede Monero. O Monero utiliza um algoritmo de mineração chamado CryptoNight, que é projetado para ser resistente a ASICs (Application-Specific Integrated Circuits), tornando a mineração mais acessível e distribuída.
Prova de Trabalho (PoW): O Monero utiliza um sistema de prova de trabalho para validar transações e criar novas moedas. Os mineradores resolvem problemas computacionais complexos usando seus dispositivos de mineração para competir pela recompensa de bloco.

####>>>> Dados Estatísticos

>- Valor de Fechamento

In [91]:

# Crie um novo DataFrame contendo apenas os dados do Monero
df_monero = df_cripto[df_cripto['Nome'] == 'Monero'].copy()

# Visualize as primeiras linhas do novo DataFrame para verificar
print(df_monero.head())


        Nome Sigla        Data    Dia_semana   Max_Dia   Min_Dia  \
3416  Monero   XMR  07/08/2018   Terça-feira  117.8030  108.0390   
3878  Monero   XMR  08/08/2018  Quarta-feira  108.0380   93.4016   
3944  Monero   XMR  09/08/2018  Quinta-feira  101.9450   94.7215   
4504  Monero   XMR  10/08/2018   Sexta-feira  101.1640   90.8204   
4570  Monero   XMR  11/08/2018        Sábado   96.9932   88.0902   

      Valor_abertura  Valor_fechamento  Volume_negociacao  Valor_Mercado  
3416        114.0930          108.0390         24304400.0   1.757439e+09  
3878        108.0380           94.5341         31289400.0   1.537758e+09  
3944         94.7215          100.5330         24062700.0   1.635341e+09  
4504        100.4730           91.7215         23751500.0   1.492007e+09  
4570         92.1754           93.6188         30488100.0   1.522869e+09  


>- Valor de Fechamento

In [92]:
# Transforme a coluna 'Close' em uma string
df_monero['Valor_fechamento'].astype(str).describe()


count        2602
unique       2602
top       108.039
freq            1
Name: Valor_fechamento, dtype: object

>--> No df_monero foram registrados 2602 valores, com 2602 valores diferentes

In [93]:
df_monero['Valor_fechamento'].describe()

count    2602.000000
mean       74.134773
std        91.180556
min         0.223522
25%         1.588685
50%        51.337759
75%       102.915250
max       483.583612
Name: Valor_fechamento, dtype: float64

![descrição da imagem, dashboard construindo em powerbi sobre a moeda bitcoin com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/monero1.png?raw=true)

>- Média de valor de fechamento teve um aumento de 15.926% e média de valor de abertura teve um aumento de 15.878%, aumentaram em tendência entre 2014 e 2021
>- As maiores médias do valor minímo e do valor máximo foram em 2021(234,64 e 262,21).

> Insight:

> O Monero teve um desempenho notável entre 2014 e 2021, com um aumento de 15.926% na média de valor de fechamento e um aumento de 15.878% na média de valor de abertura. Esses números indicam uma tendência consistente de crescimento ao longo desse período.

>Além disso, em 2021, o Monero registrou suas maiores médias de valor mínimo (234,64) e valor máximo (262,21), sugerindo um desempenho excepcional nesse ano.

>Em resumo, o Monero teve um desempenho notável durante esse período, com um aumento significativo nos valores de abertura e fechamento e alcançando máximas recordes em 2021.

####>>>>Tendência

![descrição da imagem, dashboard construindo em powerbi sobre a moeda bitcoin com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/monero2.png?raw=true)

> Ao obeservar o gráfico identificamos que os anos em que o Monero obteve sua maior queda foi em 2019 com uma baixa de -60,43% em relação a 2018.
> As maiores altas foram em 2016  com uma alta de 651,61 % em relação a 2015 e 2017 com uma alta de 1957% em relação a 2016.( obeservação não temos os dados completos do ano de 2021). Informações mais detalhadas no dashboard.

###<font color = bluelight>>>> Polkadot

#####>>>>> O Polkadot (DOT) é uma plataforma de blockchain de próxima geração que utiliza uma arquitetura única e inovadora conhecida como “paracadeias” para permitir a interoperabilidade entre diferentes blockchains. Em relação à sua pegada de carbono, o Polkadot compartilha características semelhantes a outras criptomoedas que adotam o Proof of Stake (PoS) e o Proof of Authority (PoA) como mecanismos de consenso.


####>>>> Dados Estatísticos

>- Valor de Fechamento

In [94]:

# Crie um novo DataFrame contendo apenas os dados do Polkadot
df_polkadot = df_cripto[df_cripto['Nome'] == 'Polkadot'].copy()

# Visualize as primeiras linhas do novo DataFrame para verificar
print(df_polkadot.head())


           Nome Sigla        Data     Dia_semana   Max_Dia   Min_Dia  \
15732  Polkadot   DOT  21/08/2020    Sexta-feira  3.090093  2.730919   
15733  Polkadot   DOT  22/08/2020         Sábado  4.510613  2.816762   
15734  Polkadot   DOT  23/08/2020        Domingo  4.487058  3.716271   
15735  Polkadot   DOT  24/08/2020  Segunda-feira  4.783158  3.812670   
15736  Polkadot   DOT  25/08/2020    Terça-feira  5.760775  4.501223   

       Valor_abertura  Valor_fechamento  Volume_negociacao  Valor_Mercado  
15732        2.896923          2.875028       4.931314e+07            0.0  
15733        2.878570          4.484690       4.858011e+08            0.0  
15734        4.487058          3.967066       3.207495e+08            0.0  
15735        3.987413          4.602614       4.076902e+08            0.0  
15736        4.584953          5.523393       8.988475e+08            0.0  


>- Valor de Fechamento

In [95]:
# Transforme a coluna 'Close' em uma string
df_polkadot['Valor_fechamento'].astype(str).describe()


count              320
unique             320
top       2.8750282692
freq                 1
Name: Valor_fechamento, dtype: object

>--> No df_polkadot foram registrados 320 valores, com 320 valores diferentes

In [96]:
df_polkadot['Valor_fechamento'].describe()

count    320.000000
mean      18.143080
std       13.735707
min        2.875028
25%        4.883383
50%       15.979432
75%       32.742904
max       47.945940
Name: Valor_fechamento, dtype: float64

![descrição da imagem, dashboard construindo em powerbi sobre a moeda bitcoin com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/polkadot1.png?raw=true)

>- Média de valor de abertura teve um aumento de 2.893% e média de valor de fechamento teve um aumento de 2.855%, aumentaram em tendência entre 2013 e 2021
>- As maiores médias do valor minímo e do valor máximo foram em 2021(185,48 e 207,36).

> Insight:

> Polkadot é uma criptomoeda que visa a interoperabilidade entre diferentes blockchains. Desde o seu lançamento em 2020, o DOT teve um aumento substancial na média de valor de fechamento, mostrando seu potencial no ecossistema de criptomoedas.

####>>>>Tendência

![](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/polkadot2.png?raw=true)

> Ao obeservar o gráfico identificamos que os anos em que o Litecoin obteve suas maiores quedas foi em 2015 com uma baixa de -72,63% em relação a 2014 e 2019 com uma baixa de -34,14 em relação a 2018.
> As maiores altas foram em 2017  com uma alta de 1220 % em relação a 2016 e 2021 com uma alta de 250,82% em relação a 2020.( obeservação não temos os dados completos do ano de 2021). Informações mais detalhadas no dashboard.

###<font color = bluelight>>>> Solana

#####>>>>> A Solana (SOL) é uma blockchain de alto desempenho que utiliza um mecanismo de consenso chamado “Proof of History” (PoH) em conjunto com o “Proof of Stake” (PoS). Essa abordagem visa melhorar a escalabilidade e a eficiência da rede, o que pode contribuir para uma pegada de carbono relativamente menor em comparação com blockchains baseados apenas no “Proof of Work” (PoW).

####>>>> Dados Estatísticos

>- Valor de Fechamento

In [97]:

# Crie um novo DataFrame contendo apenas os dados do Solana
df_solana = df_cripto[df_cripto['Nome'] == 'Solana'].copy()

# Visualize as primeiras linhas do novo DataFrame para verificar
print(df_solana.head())


         Nome Sigla        Data     Dia_semana   Max_Dia   Min_Dia  \
16053  Solana   SOL  11/04/2020         Sábado  1.049073  0.765020   
16054  Solana   SOL  12/04/2020        Domingo  0.956670  0.762426   
16055  Solana   SOL  13/04/2020  Segunda-feira  0.891603  0.773976   
16056  Solana   SOL  14/04/2020    Terça-feira  0.796472  0.628169   
16057  Solana   SOL  15/04/2020   Quarta-feira  0.704964  0.621531   

       Valor_abertura  Valor_fechamento  Volume_negociacao  Valor_Mercado  
16053        0.951054          0.776819       4.386244e+07            0.0  
16054        0.785448          0.882507       3.873690e+07            0.0  
16055        0.890760          0.777832       1.821129e+07            0.0  
16056        0.777832          0.661925       1.674761e+07            0.0  
16057        0.669289          0.646651       1.307528e+07            0.0  


>- Valor de Fechamento

In [98]:
# Transforme a coluna 'Close' em uma string
df_solana['Valor_fechamento'].astype(str).describe()


count                452
unique               452
top       0.776818582977
freq                   1
Name: Valor_fechamento, dtype: object

>--> No df_solana foram registrados 452 valores, com 452 valores diferentes

In [99]:
df_solana['Valor_fechamento'].describe()

count    452.000000
mean      10.471388
std       14.114444
min        0.515273
25%        1.424761
50%        2.881389
75%       14.362851
max       55.910960
Name: Valor_fechamento, dtype: float64

![descrição da imagem, dashboard construindo em powerbi sobre a moeda bitcoin com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/solana1.png?raw=true)

> Insight:

> Solana, com sua abordagem de escalabilidade, ganhou destaque rapidamente. Desde seu lançamento em 2020, o SOL viu um crescimento significativo na média de valor de fechamento, refletindo sua inovação tecnológica.

####>>>>Tendência

![descrição da imagem, dashboard construindo em powerbi sobre a moeda bitcoin com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/solana2.png?raw=true)

###<font color = bluelight>>>> Stellar

#####>>>>> A Stellar (XLM) é uma blockchain que utiliza o mecanismo de consenso conhecido como “Federated Byzantine Agreement” (FBA). Esta abordagem é considerada eficiente em termos de energia, o que resulta em uma pegada de carbono relativamente baixa em comparação com blockchains que dependem do “Proof of Work” (PoW).

####>>>> Dados Estatísticos

>- Valor de Fechamento

In [100]:

# Crie um novo DataFrame contendo apenas os dados do Stellar
df_stellar = df_cripto[df_cripto['Nome'] == 'Stellar'].copy()

# Visualize as primeiras linhas do novo DataFrame para verificar
print(df_stellar.head())


          Nome Sigla        Data    Dia_semana   Max_Dia   Min_Dia  \
14858  Stellar   XLM  18/08/2018        Sábado  0.245270  0.220600   
14924  Stellar   XLM  21/08/2018   Terça-feira  0.227195  0.210588   
14990  Stellar   XLM  22/08/2018  Quarta-feira  0.238291  0.205148   
15056  Stellar   XLM  23/08/2018  Quinta-feira  0.222162  0.206242   
15122  Stellar   XLM  24/08/2018   Sexta-feira  0.225470  0.211178   

       Valor_abertura  Valor_fechamento  Volume_negociacao  Valor_Mercado  
14858        0.239124          0.224235         73536300.0   4.209285e+09  
14924        0.217877          0.214708         57619800.0   4.030698e+09  
14990        0.219224          0.206896         57593700.0   3.884044e+09  
15056        0.213642          0.222162         50407300.0   4.170633e+09  
15122        0.214871          0.219806         46565700.0   4.126404e+09  


>- Valor de Fechamento

In [101]:
# Transforme a coluna 'Close' em uma string
df_stellar['Valor_fechamento'].astype(str).describe()


count         2527
unique        2525
top       0.216032
freq             2
Name: Valor_fechamento, dtype: object

>--> No df_stellar foram registrados 2527 valores, com 2525 valores diferentes

In [102]:
df_stellar['Valor_fechamento'].describe()

count    2527.000000
mean        0.101509
std         0.141125
min         0.001357
25%         0.002449
50%         0.049441
75%         0.124355
max         0.896227
Name: Valor_fechamento, dtype: float64

![descrição da imagem, dashboard construindo em powerbi sobre a moeda bitcoin com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/stellar1.png?raw=true)

>- Média de valor de fechamento teve um aumento de 15.926% e média de valor de abertura teve um aumento de 15.878%, aumentaram em tendência entre 2014 e 2021
>- As maiores médias do valor minímo e do valor máximo foram em 2021(234,64 e 262,21).

> Insight:

> O Stellar, focado em soluções financeiras, mostrou um desempenho sólido. De 2014 a 2021, o XLM experimentou um aumento de cerca de 3.000% na média de valor de fechamento, indicando seu papel no setor financeiro descentralizado.

####>>>>Tendência

![descrição da imagem, dashboard construindo em powerbi sobre a moeda bitcoin com análises estatísticas](https://github.com/grandpa44997/Squad6M5ResiliabyiFood/blob/main/dashimg/stellar2.png?raw=true)

> Ao obeservar o gráfico identificamos que os anos em que o Stellar obteve sua maior queda foi em 2018 com uma queda de -60,43% em relação a primeira semana de 2018.
> As maiores altas foram em 2018  com uma alta de 651,61 % em relação a 2017 e 2021 com uma alta de 357% em relação a 2020.( obeservação não temos os dados completos do ano de 2021). Informações mais detalhadas no dashboard.

## <font color = bluelight>>> Dados Prontos para Análise Descritiva


>- Valor de Fechamento

In [103]:
# Transforme a coluna 'Close' em uma string
df_cripto['Valor_fechamento'].astype(str).describe()


count     19022
unique    18994
top       104.0
freq          3
Name: Valor_fechamento, dtype: object

>--> No df_cripto foram registrados 19022 valores, com 18994 valores diferentes
o valor mais alto foi 104.0

In [104]:
df_cripto['Valor_fechamento'].describe()

count    19022.000000
mean      1117.648700
std       5095.065126
min          0.000087
25%          0.073516
50%          4.466921
75%        152.225020
max      63503.457930
Name: Valor_fechamento, dtype: float64

>- A maior média de falor de fechameto foi em Maio com  0.5137 obtendo um um percentual de variação de 732 (%) a mais do que Outubro o mês que teve a menor média de valor de fechamento com ($) 0.0617